website/docs/development/release.md

```diff
@@ -6,17 +6,26 @@ sidebar_label: Release process
 
 The release process may be automated in the future.
 
+The versioning strategy is that the repo is configured to be on the next version and the version should be bumped AFTER the release, not before.
+
+One should check that the version is the correct one against pypi before releasing.
+
+### Hydra Release Process
+
 - Checkout the release branch.
-- Update the version in `hydra/__init__.py`.
+- Check the version in `hydra/__init__.py`. This should be one version higher than the version available on PyPI. If it is the same version as PyPI, bump the version locally.
```

Comment:
When do you commit the updated news file?

Reply:
Should be the last step, after bumping the version. Will update

Reply:
Added in https://github.com/facebookresearch/hydra/pull/1385/commits/e63c9a6f189696331b7a1c19e7ef8e1e4064699d